### PR TITLE
Improve tests 

### PR DESCRIPTION
**Pull request by mymas**
Before submitting your pull request, check whether your code adheres to PHPMailer
coding standards by running the following command:

`./vendor/bin/php-cs-fixer --diff --dry-run --verbose fix `

And committing eventual changes. It's important that this command uses the specific version of php-cs-fixer configured for PHPMailer, so run `composer install` within the PHPMailer folder to use the exact version it needs.

### CODE REVIEW

**Comment by Synchro** [comment]
This is awesome, thank you! I will review it tomorrow.

**Comment by mymas** [comment]
@Synchro 

I'm sorry for fix codes soon and I did it as you reviewed.

Please review one more when you have time.

File: test/PHPMailerTest.php [hunk 1]
```diff
@@ -2454,30 +2525,59 @@ public function testPopBeforeSmtpBad()
...
+        // $this->Mail->smtpClose();
+
         $this->Mail->Host = ' localhost:12345 ; ' . $_REQUEST['mail_host'] . ' ';
-        $this->assertTrue($this->Mail->smtpConnect(), 'SMTP hosts with stray spaces failed');
+        $this->assertFalse($this->Mail->smtpConnect(), 'SMTP hosts with stray spaces failed');
         $this->Mail->smtpClose();
+
+        // Need to pick a harmless option so as not cause problems of its own! socket:bind doesn't work with Travis-CI
         $this->Mail->Host = $_REQUEST['mail_host'];
-        //Need to pick a harmless option so as not cause problems of its own! socket:bind doesn't work with Travis-CI
```
[the remark this answers is not in the document]
Comment (mymas) [reply test/PHPMailerTest.php]: Thanks for your review.
This is just my mistake. 

I'm sorry and I fixed it.

File: test/PHPMailerTest.php [hunk 2]
```diff
@@ -2454,30 +2525,59 @@ public function testPopBeforeSmtpBad()
...
-        );
+        $this->assertFalse($this->Mail->smtpConnect(['ssl' => ['verify_depth' => 10]]));
+        
+        $this->Smtp = $this->Mail->getSMTPInstance();
+        $this->assertInstanceOf( get_class($this->Smtp) , $this->Mail->setSMTPInstance($this->Smtp));
+        $this->assertFalse($this->Smtp->startTLS(), 'SMTP connect with options failed');
+        $this->assertFalse($this->Mail->SMTPAuth);
+        $this->Mail->smtpClose();
+    }
+    public function testOAuth()
```
[the remark this answers is not in the document]
Comment (mymas) [reply test/PHPMailerTest.php]: Thanks for your review and I added dockblock.

File: test/PHPMailerTest.php [hunk 3]
```diff
@@ -2454,30 +2525,59 @@ public function testPopBeforeSmtpBad()
...
+        );
+        
+        $oauth = new OAuth($options);
+        $this->assertInstanceOf(OAuth::class, $oauth);
+        $subject = $PHPMailer->setOAuth($oauth);
+        $this->assertNull($subject);
+        $this->assertInstanceOf(OAuth::class,$PHPMailer->getOAuth());
     }
 }
-
```
[the remark this answers is not in the document]
Comment (mymas) [reply test/PHPMailerTest.php]: Thanks for your review.
I forgot code style and I fixed it.

File: test/PHPMailerTest.php [hunk 4]
```diff
@@ -16,6 +16,7 @@
 use PHPMailer\PHPMailer\POP3;
 use PHPUnit\Framework\TestCase;
 
+require('/var/www/html/vendor/autoload.php');
```
[the remark this answers is not in the document]
Comment (mymas) [reply test/PHPMailerTest.php]: I'm sorry. This is mistake and I remove this line.

**Comment by Synchro** [comment]
Code style is all OK, but now it has a real test failure!

**Comment by Synchro** [comment]
Looks like there's just a bit of whitespace on a blank line.

**Comment by mymas** [comment]
I'm fix this a little bug. 
please wait.

**Comment by mymas** [comment]
@Synchro 
Thank you for your advice. I understood  coding standard and Travis-CI's trouble shooting and I should have asked you about these topics when I started this. This OSS gave me precious experience in my develop software. Thank you for your kindness!

**Pull request merged**